routes/web.php

```diff
@@ -1,9 +1,28 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
+use Lightit\Backoffice\Employee\App\Controllers\ListEmployeesController;
 use Lightit\Shared\App\Exceptions\InvalidActionException;
+use Lightit\Backoffice\Employee\App\Controllers\StoreEmployeeController;
+use Lightit\Backoffice\Task\App\Controllers\GetTaskController;
+use Lightit\Backoffice\Task\App\Controllers\ListTasksController;
+use Lightit\Backoffice\Task\App\Controllers\StoreTaskController;
+use Lightit\Backoffice\Task\App\Controllers\UpsertTaskController;
 
-Route::get('invalid', static fn() => throw new InvalidActionException("Is not valid"));
+Route::get('/', function(){
+    return view('app');
+});
+
+Route::post('/employees', StoreEmployeeController::class)->name('employees');
+
+Route::get('/employees', ListEmployeesController::class);
+
+Route::get('/tasks', ListTasksController::class);
 
-Route::get('{unknown}', static fn () => view('app  '))->where('unknown', '^(?!api).*$');
+Route::get('/tasks/{task}', GetTaskController::class);
+
+Route::post('/tasks', UpsertTaskController::class)->name('tasks');
```

Comment:
```suggestion
Route::post('/tasks', UpsertTaskController::class)->name('tasks.store');

Route::put('/tasks/{task}', UpsertTaskController::class)->name('tasks.update');
```
You should have a route for creating and a route for updating.